README.md

```diff
@@ -111,6 +111,13 @@ on django.conf.settings with the required format to connect to your redis server
     REDIS_URL = redis://localhost:6370
 ```
 
+The cache healthcheck tries to write and read a specific key within the cache backend.
+It can be customized by setting `HEALTHCHECK_CACHE_KEY` to another value:
+
+```python
+    HEALTHCHECK_CACHE_KEY = custom_healthcheck_key
```

Comment:
```suggestion
    HEALTHCHECK_CACHE_KEY = 'custom_healthcheck_key'
```

Reply:
@skarzi For both suggestions I was about to add those quotes around them, but as they was also left out from the other settings, I decided to leave them as they are.

Maybe this would be another refactoring PR after merging this one? In this way we could fix the other settings, too.

Reply:
ok, so then it makes sense to add these quotes in a separate PR 👍

Reply:
@skarzi What are the next steps here? Do you resolve the discussions in case you're fine with my decisions or am I allowed to do that? In my opinion all threads can be resolved/closed.